File: src/main/java/com/itacon/billing/invoiceservice/model/Address.java
package com.itacon.billing.invoiceservice.model;

import javax.persistence.Embeddable;

import lombok.Data;
import lombok.NoArgsConstructor;

@Embeddable
@Data
@NoArgsConstructor
public class Address {
	private String street;
	private String number;
	private String city;
	private String zipcode;
	private String country;
	private String district;
	private String description;
}

File: src/main/frontend/src/types/Address.ts
export interface Address {
    street: string
    number?: string
    zipcode?: string
    city: string
    country: string
    district?: string
    description?: string
}

export const AddressInit: Address = {
    street: '',
    number: '',
    zipcode: '',
    city: '',
    country: 'Guiné-Bissau',
    district: '',
    description: ''
}